src/components/Forms/CreateNewServiceForm.jsx

```diff
@@ -122,18 +122,18 @@ export default function CreateNewServiceForm() {
 					Tax Key
 					<StyledFieldset>
 						<label>
-							<input value="0%" name="taxKey" type="radio" {...register('taxKey')} />
+							<input value={'0'} name="taxKey" type="radio" {...register('taxKey')} />
```

Comment:
Since you can just pass the value as a number, you don't need the curly braces here:
```suggestion
							<input value='0' name="taxKey" type="radio" {...register('taxKey')} />
```